lib/cartopy/tests/mpl/test_pseudo_color.py

```diff
@@ -30,18 +30,6 @@
 from cartopy.tests.mpl import MPL_VERSION
 
 
-def test_pcolormesh_fully_masked():
```

Comment:
We are now plotting the entire mesh regardless, so that we can have the vertices for future `set_array` calls. I don't think this is a huge issue because someone can just check for fully masked data themselves beforehand and not issue the plot command if that is what they desire.